src/main/java/cobook/buddywisdom/global/util/MessageUtil.java

```diff
@@ -1,5 +1,7 @@
 package cobook.buddywisdom.global.util;
 
+import static cobook.buddywisdom.global.vo.MessageTemplate.*;
```

Comment:
```suggestion
import static cobook.buddywisdom.global.vo.MessageTemplate.DATETIME_FORMAT;
```

사용하는것만 가져오는게 좋을것 같아요.

see
- https://dev.to/kingori/import-5531
- https://www.youtube.com/watch?v=RVO02Z1dLF8
- https://invicr.github.io/2020/03/03/IntelliJ-Import-%EC%84%A4%EC%A0%95/

Reply:
넵 참고하겠습니다! 감사합니다!